Tutorials/intro-to-octave/set_octave.py

```diff
@@ -134,10 +134,11 @@ def octave_settings(qmm, qm, prog, config, octave_config, external_clock=False,
         # Change to the relevant external frequency
         qmm.octave_manager.set_clock("octave1", ClockType.External, ClockFrequency.MHZ_10)
         # If using a clock from the OPT, use this command instead
-        # qmm.octave_manager.set_clock(octave, ClockType.Buffered, ClockFrequency.MHZ_1000)
+        # qmm.octave_manager.set_clock("octave1", ClockType.Buffered, ClockFrequency.MHZ_1000)
     else:
         qmm.octave_manager.set_clock("octave1", ClockType.Internal, ClockFrequency.MHZ_10)
```

Comment:
Maybe change all of these to the new format?
`qm.octave.set_clock("octave1", clock_mode=ClockMode.External_10MHz)`

Reply:
I tried to do it now and I don't see it in the SDK...